portfolio/style.css

```diff
@@ -0,0 +1,148 @@
+body {
+  font-family: 'Syncopate', sans-serif;
+}
+
+.container-fluid {
+  padding-left: 0px !important;
+  padding-right: 0px !important;
+}
+
+.nav {
+  background: black;
+  position: fixed;
+  top: 0px;
+  width: 100%;
+}
+
+.nav-pills {
+  background: black;
+}
+
+.nav-link {
+  color: white;
+}
+
+.btn {
+  color: black;
+}
+
+.fa-github {
+  color: #b36b00;
+}
+
+.fa-twitter {
+  color: #0066ff;
+}
+
+.fa-facebook-official {
+  color: #006600;
+}
+
+.btn-circle {
+  width: 50px;
+  height: 50px;
+  margin-top: 10px;
+  padding: 5px 10px;
+  border: 2px solid white;
+  border-radius: 100% !important;
+  font-size: 30px;
+  color: white;
+  background: transparent;
+  -webkit-transition: background 0.3s ease-in-out;
+  -moz-transition: background 0.3s ease-in-out;
+  transition: background 0.3s ease-in-out;
+}
+
+.btn-circle:hover,
+.btn-circle:focus {
+  outline: none;
+  color: white;
+  background: rgba(255, 255, 255, 0.1);
+}
+
+h1 {
+  padding: 35px 0px 15px 0px;
+  margin: 0px 0px 10px 0px;
+  font-size: 8vw;
+  font-family: 'Lobster', sans-serif;
+  width: 50%;
+  margin-right: auto;
+  margin-left: auto;
+  border-radius: 3%;
```

Comment:
Is this a left over from a previous design iteration? If it is, remove it to keep things clean and prevent unexpected behaviour. CSS is already messy as it is so keeping it tidy is vital, trust me :)